ToyBox/classes/MonkeyPatchin/BagOfPatches/LevelUp.cs

```diff
@@ -431,5 +432,76 @@ public static class MultiplyFeatPoints_LevelUpHelper_AddFeatures_Patch {
                 return false;
             }
         }
+
+
+        [HarmonyPatch(typeof(ApplyClassProgression), "ApplyProgressionLevel")]
```

Comment:
Is there a particular reason you would want your level ups to go 
`1 2 3 ... 18 19 20 18 19 18 19 18 19 18 19 19 18 18 19 18 19 18 19 18 19 19 18 18 19 18 19 18 19 18 19 19 18 18 19 18 19 18 19 18 19 19 20` 
given that most classes have special abilities that advance outside of that? Wizard for example is a bonus feat every 5 levels, and barbarian damage reduction goes from 1/4 levels to 1/2 levels. I personally suspect this is better as its own mod (and I'm happy to make sure toybox doesn't step on you) and actually modify the blueprints to get their proper advancement.

Reply:
I first tried to repeat levels 2-19, but found that this would lead to some bugs. For example, you will be forced to choose another animal partner，Reducing the interval will sometimes lead to the situation that there is no ability to choose and upgrade. 18-19 is a safer choice I explored。